Documentation/guides/OneDotNetEmbeddedResources.md

```diff
@@ -0,0 +1,268 @@
+# [spec] Embedded Resources and `.nupkg` Files
+
+Traditionally, a Xamarin.Android class library can contain many types
+of Android-specific files:
+
+* `@(AndroidAsset)` files in `Assets\`
+* `@(AndroidEnvironment)` text files
+* `@(AndroidJavaLibrary)` `.jar` files
+* `@(AndroidNativeLibrary)` files in `lib\[arch]\*.so`
```

Comment:
Missing `EmbeddedNativeLibrary` used for class libraries

Reply:
I'll list `@(EmbeddedNativeLibrary)`.

We are aspiring to change all these to:
```xml
<!-- Include and bind -->
<AndroidLibrary Include="foo.aar" Bind="true" />
<!-- Just include, do not bind -->
<AndroidLibrary Include="bar.aar" />
<AndroidLibrary Include="bar.jar" />
<!-- Native libraries need ABI directory -->
<AndroidLibrary Include="armeabi-v7a\libfoo.so" />
<AndroidLibrary Include="x86\libfoo.so" />
```

Reply:
I like that change - much simpler to have a single build action for these.

Reply:
Should this support an attribute to explicitly specify ABI for .so's?

Reply:
> an attribute to explicitly specify ABI

I thought about this, but maybe it should just be the directory name?

How many times would people need to do something like this:
```
<AndroidLibrary Include="foo\libFoo.so" Abi="armeabi-v7a" />
<AndroidLibrary Include="bar\libFoo.so" Abi="x86" />
```

Native libraries will likely all have the same file name, too, or `[DllImport]` gets complicated.